### PR TITLE
[FrameworkBundle] Config reference for Secrets

### PR DESCRIPTION
**Pull request by Nyholm**

### CODE REVIEW

File: reference/configuration/framework.rst [hunk 1]
```diff
@@ -197,6 +197,12 @@ Configuration
   * `utf8`_
 
 * `secret`_
+* `secrets`_
+
+  * `vault_directory`_
+  * `local_dotenv_file`_
+  * `decryption_env_var`_
```
Comment (OskarStark) [inline reference/configuration/framework.rst:202]: Shall we sort them alphabetically?

File: reference/configuration/framework.rst [hunk 2]
```diff
@@ -2483,6 +2489,31 @@ annotation changes). For performance reasons, it is recommended to disable
...
+**type**: ``string`` **default**: ``%kernel.project_dir%/.env.%kernel.environment%.local``
+
+Path to an dotenv file that holds secrets. This is primarily used for testing.
+
+vault_directory
+...............
+
+**type**: ``string`` **default**: ``%kernel.project_dir%/config/secrets/%kernel.environment%``
+
+The directory where the secrets is stored.
```
Comment (OskarStark) [inline reference/configuration/framework.rst]:
```suggestion
The directory where the secrets file is stored.
```

File: reference/configuration/framework.rst (hunk 1 above)
Comment (Nyholm) [reply reference/configuration/framework.rst:202]: Yes sir!

File: reference/configuration/framework.rst (hunk 2 above)
Comment (Nyholm) [reply reference/configuration/framework.rst]: Hm.. It is just not one file. I updated this line

Comment (OskarStark) [reply reference/configuration/framework.rst]: Perfect 👌🏻

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Thank you Tobias.

**Comment by Nyholm** [comment]
Thank you for merging

**Pull request merged**